buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -237,6 +238,7 @@ SHORT_NAMES = ['all' : ['ppc64le_linux','s390x_linux','x86-64_linux','ppc64_aix'
             'alinux64xluma' : ['aarch64_linux_xl_uma'],
             'alinux64largeheap' : ['aarch64_linux_xl'],
             'alinux64mxd' : ['aarch64_linux_mixed'],
+            'amac' : ['aarch64_mac'],
```

Comment:
Can we strive for consistency in the trigger names across architectures?  `amac` (or even `amacos`) is fine, but I think there should be a family of `xmac` triggers for x86 as well.  `osx` is an antiquated name (since 2016) and should probably start to be phased out.

Reply:
Agree we need to adjust the existing triggers. Luckily, we can add new ones on top of the old ones while we phase them out. Unless we prefer to do a hard change over.

Reply:
@0xdaryl cc @pshipton 
I'll add `amac` here, and also add `xmac` and corresponding extensions of it as well that are still active. We can have multiple short-name triggers map to the same specs. And we can phase out osx over time if we feel the need. I don't think we need amac variants for the others at the moment (aot, valhalla etc).